### PR TITLE
Minor JavaDoc fix in EntryOperation

### PR DESCRIPTION
**Pull request by Holmistr**
Minor JavaDoc typo fix.

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryOperation.java [hunk 1]
```diff
@@ -85,10 +85,10 @@
  * If the EntryProcessor implements Offloadable the invocation scenario looks as follows:
  * - EntryOperation fetches the entry and locks the given key on partition-thread
  * - Then the processing is offloaded to the given executor
- * - When the processing finishes
+ * - Then the processing finishes
```
Comment (Holmistr) [inline hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryOperation.java:88]: Is this correct or was it a beginning of a new sentence just with improper padding?

Comment (ahmetmircik) [reply hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryOperation.java:88]: `When` seems more appropriate here, also having this as a html list item can be more clear.

**Review by ahmetmircik — CHANGES_REQUESTED** [review]
comments

**Comment by AyberkSorgun** [comment]
Addressed the review comment along with @Serdaro

**Review by ahmetmircik — APPROVED** [review]

**Pull request merged**